test/python/pulse/test_block.py

```diff
@@ -379,7 +379,9 @@ def test_execute_block(self):
             pulse.acquire(50, pulse.AcquireChannel(0), pulse.MemorySlot(0))
 
         backend = FakeArmonk()
-        test_result = backend.run(sched_block).result()
+        # TODO: Rewrite test to simulate with qiskit-dynamics
+        with self.self.assertWarns(DeprecationWarning):
```

Comment:
```suggestion
        with self.assertWarns(DeprecationWarning):
```

Reply:
Sigh, I fixed that locally and forgot to update the commit before pushing. It should be fixed now.